src/explore-education-statistics-admin/src/pages/publication/PublicationDetailsPage.tsx

```diff
@@ -70,7 +62,7 @@ const PublicationDetailsPage = () => {
                 : 'This publication is not archived'}
             </SummaryListItem>
           </SummaryList>
-          {permissions.canUpdatePublication && (
+          {permissions?.canUpdatePublication && (
```

Comment:
As mentioned in `PublicationContact`, we should be working with a `PublicationWithPermissions` so none of the `permissions` should be optional. Same for below this as well.